src/core.ts

```diff
@@ -186,7 +202,10 @@ function getFunctionArguments(fnDef: FunctionDefinition): Argument[] {
 }
 
 function getType(varDecl: VariableDeclaration, location: StorageLocation = varDecl.storageLocation): string {
-  const { typeString, typeIdentifier } = varDecl.typeDescriptions;
```

Comment:
`varDecl.typeDescriptions`  didn't include the `_ptr` suffix, but `varDecl.typeName.typeDescriptions` did.

Reply:
What is the relevance of the `_ptr` suffix? I don't really understand.

Reply:
What is the relevance of the `_ptr` suffix? I don't really understand.